### PR TITLE
feat: add 'Berty, Libp2p, and BLE' blog post

### PR DESCRIPTION
**Pull request by aeddi**
My first contrib :)

I really wasn't sure what tags to put, feel free to correct.

### CODE REVIEW

File: bertytech/content/blog/bluetooth-low-energy/index.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
+---
+author: aeddi
+title: Berty, Libp2p and Bluetooth Low Energy
+date: 2020-04-08
+image: "https://i.ibb.co/BNBbZxH/2020-04-06-Berty-blogpost-Bluetooth-Low-Energy.jpg"
```
Comment (pierreboc) [inline bertytech/content/blog/bluetooth-low-energy/index.md:5]: Hum pas plus simple de la mettre dans le même folder?

File: bertytech/content/blog/bluetooth-low-energy/index.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+categories:
+- announcement
+tags:
+- technical
```
Comment (pierreboc) [inline bertytech/content/blog/bluetooth-low-energy/index.md:9]: announcement aussi je dirai

File: bertytech/content/blog/bluetooth-low-energy/index.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+- **Android Nearby:** One or two months after the release of the MultiPeer Connectivity driver.
+- **BLE for Android and iOS:** No date defined and no duration estimated, but it will be the 3rd step.
+- **BLE for Linux and Windows:** Probably the wisest thing to do would be to wait for the reference versions of the drivers to be implemented (iOS and Android), but if the community is too impatient to wait for us to work on it, we won't stand in its way. :smirk:
+
+For the curious (who are definitely very curious), our new developer has also developed a test app for Multipeer Connectivity: [multipeer-connectivity-prototype](https://github.com/berty/multipeer-connectivity-prototype)
+
+I'm told in my earpiece :male-office-worker: that while I'm writing this article, he has also just successfully wired his prototype with the v1 of Berty! Here is [the link](https://github.com/berty/berty/pull/1832#issuecomment-610859687).
+So all we have to do now is to tweak it and integrate it into gomobile-ipfs.
+
+If you would like to lend a helping hand, we'd be delighted to receive assistance from our community!
```
Comment (pierreboc) [inline bertytech/content/blog/bluetooth-low-energy/index.md:221]: Link vers Community ça peut etre cool

File: bertytech/content/blog/bluetooth-low-energy/index.md (hunk 1 above)
Comment (moul) [reply bertytech/content/blog/bluetooth-low-energy/index.md]: done

File: bertytech/content/blog/bluetooth-low-energy/index.md (hunk 2 above)
Comment (moul) [reply bertytech/content/blog/bluetooth-low-energy/index.md:10]: I updated the tags, but I don't share your feeling about "announcement" for this article, maybe other people can help us choose

File: bertytech/content/blog/bluetooth-low-energy/index.md (hunk 3 above)
Comment (moul) [reply bertytech/content/blog/bluetooth-low-energy/index.md]: done

File: bertytech/content/blog/bluetooth-low-energy/index.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+### Our prognosis:
+
+- **Multipeer Connectivity:** Implementation in progress! We think and we hope that we will have it ready and available in gomobile-ipfs within a month! :tada:
+- **Android Nearby:** One or two months after the release of the MultiPeer Connectivity driver.
+- **BLE for Android and iOS:** No date defined and no duration estimated, but it will be the 3rd step.
+- **BLE for Linux and Windows:** Probably the wisest thing to do would be to wait for the reference versions of the drivers to be implemented (iOS and Android), but if the community is too impatient to wait for us to work on it, we won't stand in its way. :smirk:
+
+For the curious (who are definitely very curious), our new developer has also developed a test app for Multipeer Connectivity: [multipeer-connectivity-prototype](https://github.com/berty/multipeer-connectivity-prototype)
+
+I'm told in my earpiece :male-office-worker: that while I'm writing this article, he has also just successfully wired his prototype with the v1 of Berty! Here is [the link](https://github.com/berty/berty/pull/1832#issuecomment-610859687).
```
Comment (moul) [inline bertytech/content/blog/bluetooth-low-energy/index.md:220]:
```suggestion
I'm told in my earpiece 👨‍💼 that while I'm writing this article, he has also just successfully wired his prototype with the v1 of Berty! Here is [the link](https://github.com/berty/berty/pull/1832#issuecomment-610859687).
```

**Pull request merged**